app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -408,7 +409,7 @@ class Settings(private val appContext: Context) : PreferencesHolder {
      */
     var lastBrowseActivity by longPreference(
         appContext.getPreferenceKey(R.string.pref_key_last_browse_activity_time),
-        default = timeNowInMillis(),
+        default = 0L,
```

Comment:
👍 Might be good to add a comment since this was a bug before too technically, although it didn't matter.